src/fidesops/schemas/masking/masking_strategy_description.py

```diff
@@ -1,12 +1,13 @@
 from typing import List
 
+from fideslang.validation import FidesKey
 from pydantic import BaseModel
 
 
 class MaskingStrategyConfigurationDescription(BaseModel):
     """The description model for a specific configuration in a masking strategy"""
 
-    key: str
+    key: FidesKey
```

Comment:
This one isn't true - this is just a config to explain how masking strategies should be documented, the key here has nothing to do with a fides key.

Reply:
ah, thanks for catching this!